libraries/AP_Airspeed/AP_Airspeed.h

```diff
@@ -213,17 +226,26 @@ class AP_Airspeed
         Airspeed_Calibration calibration;
         float last_saved_ratio;
         uint8_t counter;
+        uint8_t reenable_use;
+        uint8_t airspeed_failure_basic_failed_count=0;
+        uint8_t airspeed_failure_basic_success_count=0;
+        bool    airspeed_failure_basic_renable=false;
+        uint32_t airspeed_failure_last_check=0;
+        uint32_t airspeed_failure_last_reset=0;
```

Comment:
remove the assignments, the default is already 0 but since they are usually in an array you're better off doing this in init() if it's really needed. Also, if they're all airspeed_failure then it might be easier to group them into a failure struct and drop the airspeed_failure... name.